### PR TITLE
add columns for output

### PR DESCRIPTION
**Pull request by jude253**
This PR puts the `li`s in their own divs and makes them "column style".

I also changed the color of the `li` text to be slightly grey so that we can tell these are the output words and the "updated" part of the UI.  Perhaps another color would be better. I'd love to hear feedback on that.

Also, I put the submit buttons below the 'letter color buttons' and changed the "Add Guess:" and "Choose Color:" text to "Guess:" and "Color:".

I am wondering if we should be centering the text in the "column" `div`s or maybe we could left align those and it might look better.  I think the page looks a little funny having the input area centered, but then the "column" `div`s centered, but in like like not to the center of the screen.  I wish we could keep the output list centered in the page and just tack the previous guesses off to the side, next to it, but I do not know how to do this.

@richpompeo let me know what you think of this version, I wonder if there are like too many `div`s or something like that that is not good for accessibility or if you have any other input as well!

### CODE REVIEW

**Comment by richpompeo** [comment]
For the text color change, this should be fine with current background color, it has enough color contrast. I think the structure is looking good with the changes for the Add Your Guess and Results sections. One idea/suggestion is to make the Words Left list display three columns of list items, to break up the list a bit and prevent having to scroll as far otherwise. For right now the divs are okay,  if we have a div which is user interactable then we should check about adding the proper `role` tag to it.

File: wordle-helper/src/components/Home.js [hunk 1]
```diff
@@ -498,31 +498,40 @@ const Home = () => {
...
-                    <p>Number of 5 Letter Words Left: <b>{wordList.length}</b></p>
-                    <p>Words Left:</p>
-                    <ul>
-                        {displayWordList}
-                    </ul>
+                    <div style={{ display: 'grid', gridTemplateColumns: '30% 60%', gridGap: '10%',
+                                  maxWidth: '80%', marginLeft: 'auto', marginRight: 'auto' }}>
+                        <div>
+                            <p>Previous Guesses:</p>
+                            <ul style={{ istStyle: 'none', paddingLeft: 0 }}>
```
Comment (richpompeo) [inline wordle-helper/src/components/Home.js]: should be `listStyleType`

File: wordle-helper/src/components/Home.js [hunk 2]
```diff
@@ -498,31 +498,40 @@ const Home = () => {
...
+                                {previousGuesses.map((word, index) => <Item value={word[0]} key={index}/>)}
+                            </ul>
+                        </div>
+                        <div>
+                            <p>Number of 5 Letter Words Left: <b>{wordList.length}</b></p>
+                            <p>Words Left:</p>
+                            <ul style={{ istStyle: 'none', paddingLeft: 0 }}>
```
Comment (richpompeo) [inline wordle-helper/src/components/Home.js]: should be listStyleType

Comment (richpompeo) [inline wordle-helper/src/components/Home.js]: Also see if you think adding `columns: '3 auto'` here makes sense, would look something like this
Screen Shot 2022-03-28 at 8 20 26 PM

Comment (jude253) [reply wordle-helper/src/components/Home.js]: This is super cool! I will add this! Thanks for helping me with that!

File: wordle-helper/src/components/Home.js (hunk 1 above)
Comment (jude253) [reply wordle-helper/src/components/Home.js]: Thanks bro! I am suprized `istStyle` doesn't make the page error!

File: wordle-helper/src/components/Home.js (hunk 2 above)
Comment (jude253) [reply wordle-helper/src/components/Home.js]: Oh! thanks! Oops looks like I left a typo there :/

**Comment by jude253** [comment]
> For the text color change, this should be fine with current background color, it has enough color contrast. I think the structure is looking good with the changes for the Add Your Guess and Results sections. One idea/suggestion is to make the Words Left list display three columns of list items, to break up the list a bit and prevent having to scroll as far otherwise. For right now the divs are okay, if we have a div which is user interactable then we should check about adding the proper `role` tag to it.

cool! Thanks I will make the changes you talked about.  I was thinking about changing the font color of the guesses so that we can see that those are the guesses easily as opposed to like instruction text, but there might be a better way to do that.

I don't know how to use the `role` tag, but it sounds like something I should add.  I think I will make the changes you outlined here, merge this, then make a new PR where I add the custom component for previous guesses having the letter colors!

File: wordle-helper/src/components/Home.js (hunk 2 above)
Comment (jude253) [reply wordle-helper/src/components/Home.js]: I updated this

**Comment by jude253** [comment]
@richpompeo making the changes you suggested, and I also centered the "words left" div in the page as well

**Pull request merged**